deployment-manager/pom.xml

```diff
@@ -1,62 +1,62 @@
-<?xml version="1.0" encoding="UTF-8"?>    
+<?xml version="1.0" encoding="UTF-8"?>  
 <!--
- Copyright 2016 Google Inc.
```

Comment:
Sorry if this wasn't clear, but as of 2019 the "Inc." has changed to "LLC"

Reply:
Ah, okay! Fixed!